packages/sdk/src/models/index.ts

```diff
@@ -139,19 +140,25 @@ export default class Models {
    * Fetches data from Zeitgeist and IPFS for a market with a given identifier.
    * @param marketId The unique identifier for the market you want to fetch.
    */
-  async fetchMarketData(marketId: MarketId): Promise<Market> {
+  async fetchMarketData(
+    marketId: MarketId,
+    opts?:{ address: string },
+  ): Promise<Market> {
     const ipfs = initIpfs();
 
 
     const marketRaw =
       await this.api.query.predictionMarkets.markets(marketId);
 
+    // TODO: type
```

Comment:
please create an issue to track todos and put them in the code

```suggestion
    // TODO: type (#???)
```